nucliadb/nucliadb/tests/test_search.py

```diff
@@ -476,3 +480,251 @@ async def inject_resource_with_a_sentence(knowledgebox, writer):
     bm.field_vectors.append(ev)
 
     await inject_message(writer, bm)
+
+
+@pytest.mark.asyncio
+async def test_search_relations(
+    nucliadb_reader: AsyncClient,
+    nucliadb_writer: AsyncClient,
+    nucliadb_grpc: WriterStub,
+    knowledgebox,
+):
+    predict_mock = Mock()
+    set_utility(Utility.PREDICT, predict_mock)
+
+    resp = await nucliadb_writer.post(
+        f"/kb/{knowledgebox}/resources",
+        headers={"X-SYNCHRONOUS": "True"},
+        json={
+            "title": "My resource",
+            "slug": "myresource",
+            "summary": "Some summary",
+        },
+    )
+    assert resp.status_code == 201
+    rid = resp.json()["uuid"]
+
+    relation_nodes = {
```

Comment:
NIT: would be nice to have this in a fixture to use in future tests

Reply:
Agree, moving it to a fixture